styles.css

```diff
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --font-family-body: 'Poppins', sans-serif;
+  --ff-headings: 'Playfair Display', serif;
+  --font-size-h1: clamp(1.5rem, -0.25rem + 20vw, 6rem);
+  --gap-header: clamp(12rem, 100vw, 25rem);
+  --button-SC-width: min(90vw, 700px);
+  --adaptive-sizing-section-2: min(100vw, 600px);
+  --fs-body: clamp(1rem, 0.3846153846153846rem + 3.076923076923077vw, 1.5rem);
+}
+
+header {
+  display: flex;
+  gap: 60vw;
+  margin-bottom: 2rem;
+}
+
+.section-1 {
+  position: relative;
+}
+
```

Comment:
- [ ] Kindly align this section using flexbox based on the requirements for this project. Using flexbox gives a lot of options to align all our elements, plus, it's responsive. 💻

Reply:
it says nowhere that i have to use flexbox to align a section i created i used flex box for my header and my page is respnsive up to 700px because the plan is to add a media query for my desktop making it a seamless this is another instance i do not see your motivation  @Gambit142

Reply:
As mentioned in the project requirements, please style the `header` and the `headline` using Flexbox.
![image](https://user-images.githubusercontent.com/83596135/212432721-f37c80ef-eec1-42fc-a6e2-900cd78d9767.png)